### ISSUE TITLE
Checkbox field id has dash appended

### ISSUE THREAD

**Issue by hughbris** [getgrav/grav-plugin-form#421]
I set the `id` property for a checkbox field in frontmatter, The field is being output with '-' appended to that id string. (Makes it difficult to select using JS DOM methods, affects label accessibility, etc.)

Looks like the [checkbox template includes reference to a `key` Twig variable](https://github.com/getgrav/grav-plugin-form/blob/a6382113000a60f45c6587d46b9cebcb2c9fb426/templates/forms/fields/checkbox/checkbox.html.twig#L7). Couldn't figure out where it's meant to be coming from, but it's empty in my case.

My guess is that that line
```twig
{% set id = field.id|default(field.name) ~ '-' ~ key %}
```
 should be
```twig
{% set id = field.id|default(field.name ~ '-' ~ key) %}
```
or
```
{% set id = field.id|default(field.name) ~ (key ? '-' ~ key) %}
```
depending on what the intent and context is here. If you advise me, I'll make a PR.

Not sure if other field types are affected.

**Comment by rhukster** [comment]
I think this one: `{% set id = field.id|default(field.name ~ '-' ~ key) %}` is the intended approach. although key should be in index of the looping over the elements.. this is to ensure they are unique. let me take a look...

**Comment by rhukster** [comment]
Ok, i see what happened here.  We added the key logic to `checkboxes` and it was copied to `checkbox` which is the singular variation.  I've removed the `-key` logic completely for next release in this case.

**Comment by hughbris** [comment]
Fixed by eea1e45, cross referencing and closing.
